src/environment-data-provider.test.ts

```diff
@@ -21,14 +24,35 @@ describe('environment-data-provider', () => {
                 inapplicable: [],
             };
 
-            const stubEnvironmentData: EnvironmentData = {
+            const expectedResults: EnvironmentData = {
                 timestamp: stubTimestamp,
                 targetPageUrl: stubTargetPageUrl,
                 targetPageTitle: stubTargetPageTitle,
             };
 
-            const environmentDataResults = getEnvironmentData(stubAxeResults);
-            expect(environmentDataResults).toEqual(stubEnvironmentData);
+            const actualResults = getEnvironmentDataFromResults(stubAxeResults);
+            expect(actualResults).toEqual(expectedResults);
+        });
+    });
+
+    describe('getEnvironmentDataFromEnvironment', () => {
+        it('returns an EnvironmentData object from environment data extracted from the environment', () => {
+            const actualResults = getEnvironmentDataFromEnvironment();
+            expect(actualResults).toHaveProperty('timestamp');
+            expect(actualResults).toHaveProperty('targetPageUrl');
+            expect(actualResults).toHaveProperty('targetPageTitle');
+        });
```

Comment:
For readability purposes, it may help to separate with new lines between test cases.